Initial react app

CradleKing24 opened the pull request.

CradleKing24 commented:
Work in progress, attempting to setup github actions and deploy the application as a github page.  [Tutorial](https://codeburst.io/deploying-a-react-app-using-github-pages-and-github-actions-7fc14d380796)

ellefjus000 commented:
@CradleKing24 1 cOmMiT fOr EaCh tHiNg .... Jk

I need to adjust my notification setting because im getting blasted

CradleKing24 commented:
@ellefjus000 @mpmcd1 take a look at this when you get a chance/meet.  I added github actions to automatically deploy our application changes to github pages to start with so we could test stuff easier/more quickly.

Right now both the `env/test` branch and the `master` branch will deploy the application when merged into.  PR's off of `master` and `env/test` will kick-off a build only.

Added URL to the page in the README.  some links dont work but its at least a start.

Can merge whenever or can keep open for now as an example to discuss later

CradleKing24 commented:
@ellefjus000  @mpmcd1 Completing the full React tutorial and completing through step "Navigating between pages" should be enough to all have a good baseline (thats as far as I have gotten).

For configuring Actions, I followed a configuration and tweaked it for our needs so nothing major to review there since I diverted from the initial copy.

ellefjus000 commented:
@CradleKing24 am i correct in that a "env/test" merge and a "master" merge will result in the same github page being built? as in there isn't a github page for test and another for production?

CradleKing24 commented:
@ellefjus000 yep correct!  So far both env/test and master will deploy to the same GitHub pages.  Next step would figure out a real prod and test separation
